platform-core-module/src/main/java/com/softicar/platform/core/module/component/external/license/rule/ExternalComponentLicensePredefinedRules.java

```diff
@@ -23,6 +23,11 @@ private ExternalComponentLicensePredefinedRules() {
 
 		this.list = new ArrayList<>();
 
+		// Artifact does not contain license information.
+		// License defined at:
+		// https://github.com/google/guava/blob/master/COPYING
+		addLibrary("bcprov-jdk15on", ".*", License.MIT);
```

Comment:
```suggestion
		addLibrary("bcprov-jdk15on", ".*", License.BOUNCY_CASTLE);
```
...that license enumerator needs to be created.